File: Java Codewars/8kyu/easy logs.java
/*
Description

Given a logarithm base X and two values A and B, return a sum of logratihms with the base X: 
logXA + logXB.
*/

public class EasyLogs {

  public static double logs(double x, double a, double b) {
    return (Math.log(a) + Math.log(b)) / Math.log(x);
  }
}

File: Javascript Codewars/8kyu/easy logs.js
/*
Description

Given a logarithm base X and two values A and B, return a sum of logratihms with the base X: 
logXA+logX​B.
*/

function logs(x, a, b) {
  return Math.log(a) / Math.log(x) + Math.log(b) / Math.log(x);
}
